.github/workflows/action-ros-lint.yaml

```diff
@@ -0,0 +1,22 @@
+name: ament_lint
+
+on: [pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-20.04
+    container:
+      image: ubuntu:focal
+    strategy:
+      fail-fast: false
+      matrix:
+          linter: [copyright, flake8, pep257, xmllint]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ros-tooling/setup-ros@v0.2
+      - uses: ros-tooling/action-ros-lint@0.1.3
+        with:
+          linter: ${{ matrix.linter }}
+          package-name: 
+            rexrov2
+            rexrov2_description
```

Comment:
Missing EOL (I suggest enabling the "files.insertFinalNewline" and "files.trimFinalNewlines" options in your vscode workspace)